README.md

```diff
@@ -9,11 +9,17 @@
 - You don&rsquo;t know JS by Kyle Simpson
 - Eloquent JavaScript by Marijn Haverbeke
 
-# React
+# Typescript
+
+- Instantly parse JSON into type [QuickType] (https://app.quicktype.io/)
```

Comment:
Markdown link isn't displaying properly. There shouldn't be any space between "[QuickType]" and "(https:/....)"